### PR TITLE
Challenge Prototypes

### PR DESCRIPTION
**Pull request by AlexGonRod**

### CODE REVIEW

File: course/frontend/prototypes/scripts/main.js [hunk 1]
```diff
@@ -20,4 +20,53 @@ console.log(1, 'fiestaD.brand should be ford', fiestaD.brand);
...
+    this.gender = gender;
+    this.interests = interests;
+    
+}
+var person = new Person('Alex', 'Gonzalez', 34, 'male', ['music', 'cinema', 'development', 'natacion'])
+Person.prototype.greeting = function () { return "I'm " + person.firstName }
+Person.prototype.farewell = function () { return person.firstName + ' has left the building. Bye for now!' }
+Person.prototype.bio = function () {
+
+    return person.firstName + ' ' + person.lastName + ' is ' + person.age + ' years old.\n' + (this.pronom = gender === 'm' ? 'he' : 'she';) + ' likes ' + person.interests.toString().replace(/,/g, ' and ')
```
Comment (AsierDev) [inline course/frontend/prototypes/scripts/main.js]: captura

Algo te falla en esta linea @AlexGonRod !

Comment (AlexGonRod) [reply course/frontend/prototypes/scripts/main.js]: Cierto, después de hacer el PR lo he arreglado..

**Comment by manuelbarzi** [comment]
WRONG state indicating "AlexGonRod  wants to merge 7 commits into manuelbarzi:develop from unknown repository" -> unknown repository? what happened here? :)

**Comment by manuelbarzi** [comment]
@AlexGonRod please, place your solution in staff/pink/prototypes folder and PR it again ,)

**Pull request closed without merging**